nav2_system_tests/src/recoveries/spin_recovery_tester.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 Intel Corporation
+// Copyright (c) 2020 Sarthak Mittal
```

Comment:
Ah, these look awfully modified from source. You should include both intel and yourself.